### PR TITLE
Profile snagging fix

### PR DESCRIPTION
**Pull request by alexxpetrov**
Closes:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1831

* icon issue was about the theme. Gone now
image

### CODE REVIEW

File: src/components/attendee/Password/ProfileModalChangePassword.module.scss [hunk 1]
```diff
@@ -1,11 +1,13 @@
+@use "scss/attendee/space";
+
 .ProfileModalChangePassword {
   &__input {
-    margin: 8px 0;
+    margin: space.$sm 0;
```
Comment (goran-peoski-work) [inline src/components/attendee/Password/ProfileModalChangePassword.module.scss]: Please use `space.gap()` and `space.button()`, just like `space.empty()` to add semantic meaning / uniformity. Wouldn't be against of adding one more `space` function if we need for some common thing

File: src/components/attendee/Password/ProfileModalChangePassword.module.scss [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
     display: flex;
     flex-direction: column;
   }
 
   &__title {
-    margin: 4px 0 8px 0;
+    margin-top: space.$xs;
```
Comment (goran-peoski-work) [inline src/components/attendee/Password/ProfileModalChangePassword.module.scss]: Same comment as above

File: src/components/attendee/ProfileOverlay/ProfileOverlay.module.scss [hunk 3]
```diff
@@ -41,3 +41,7 @@
     line-height: 1.4;
   }
 }
+
+.logOut {
+  margin-bottom: space.empty(3);
```
Comment (goran-peoski-work) [inline src/components/attendee/ProfileOverlay/ProfileOverlay.module.scss]: `space.button(1)` is effectively `space.empty(3.5)` for uniformity reasons

**Review by goran-peoski-work — COMMENTED** [review]
Looks good, but I do  think we can replace hard-coded values with few functions from `space` module

**Review by goran-peoski-work — APPROVED** [review]
Approving with the idea what is quick to update from the previous review is changed before merge.

File: src/components/attendee/ProfileOverlay/ProfileOverlay.module.scss (hunk 3 above)
Comment (alexxpetrov) [reply src/components/attendee/ProfileOverlay/ProfileOverlay.module.scss]: [84cb042](https://github.com/sparkletown/sparkle/pull/3030/commits/84cb042903c42e21a52cb03d0e89e0b6aed4cdcb)

File: src/components/attendee/Password/ProfileModalChangePassword.module.scss (hunk 2 above)
Comment (alexxpetrov) [reply src/components/attendee/Password/ProfileModalChangePassword.module.scss]: [84cb042](https://github.com/sparkletown/sparkle/pull/3030/commits/84cb042903c42e21a52cb03d0e89e0b6aed4cdcb)

File: src/components/attendee/Password/ProfileModalChangePassword.module.scss (hunk 1 above)
Comment (alexxpetrov) [reply src/components/attendee/Password/ProfileModalChangePassword.module.scss]: [84cb042](https://github.com/sparkletown/sparkle/pull/3030/commits/84cb042903c42e21a52cb03d0e89e0b6aed4cdcb)

**Pull request merged**